docs/data/external-oasis-repos.md

```diff
@@ -63,94 +75,186 @@ root@host # chkconfig httpd on
 
-3.  The repository service administrator creates the repository with these command like these, where ownerid is the user id that will have write access:
+The initial repository creation must be run as `root`:
 
-        :::console
-        root@host # echo -e "\*\\t\\t-\\tnofile\\t\\t16384" >>/etc/security/limits.conf 
-        root@host # ulimit -n 16384 
-        root@host # cvmfs_server mkfs -o ownerid repo.domain.name 
-        root@host # echo "CVMFS_AUTO_TAG=false" >>/etc/cvmfs/repositories.d/repo.domain.name/server.conf 
-        root@host # (echo Order deny,allow;echo Deny from all;echo Allow from 127.0.0.1;echo Allow from ::1;echo Allow from 129.79.53.0/24;echo Allow from 2001:18e8:2:6::/56) >/srv/cvmfs/repo.domain.name/.htaccess
+    :::console
+    root@host # echo -e "\*\\t\\t-\\tnofile\\t\\t16384" >>/etc/security/limits.conf
+    root@host # ulimit -n 16384
+    root@host # cvmfs_server mkfs -o %BLUE%LIBRARIAN%ENDCOLOR% %RED%example.opensciencegrid.org%ENDCOLOR%
+    root@host # echo "CVMFS_AUTO_TAG=false" >>/etc/cvmfs/repositories.d/%RED%example.opensciencegrid.org%ENDCOLOR%/server.conf
+    root@host # (echo Order deny,allow;echo Deny from all;echo Allow from 127.0.0.1;echo Allow from ::1;echo Allow from 129.79.53.0/24;echo Allow from 2001:18e8:2:6::/56) >/srv/cvmfs/%RED%example.opensciencegrid.org%ENDCOLOR%/.htaccess
+
+Here, we increase the number of open files, create the repository using the `mkfs` command, adjust the configuration,
+and then limit the hosts that are allowed to access the repo to the OSG CDN.
 
-    If you might be hosting any hardlinks that span directories (e.g. the git package) and are using aufs (that is, EL6), also do the following:
+!!! note
+    If you might be hosting any hardlinks that span directories (e.g. publishing a `git` repository) and are using
+    EL6 with aufs, make the following configuration change:
 
         :::console
         root@host # echo "CVMFS_IGNORE_XDIR_HARDLINKS=true" >>/etc/cvmfs/repositories.d/repo.domain.name/server.conf
 
-    Verify that the repository is readable over http with the following command:
+Verify that the repository is readable over HTTP:
 
-        :::console
-        root@host # wget -qO- http://localhost:8000/cvmfs/repo.domain.name/.cvmfswhitelist|cat -v
+    :::console
+    root@host # wget -qO- http://localhost:8000/cvmfs/%RED%example.opensciencegrid.org%ENDCOLOR%/.cvmfswhitelist | cat -v
+
+That should print several lines including some gibberish at the end.
+
+Host a Repository on OASIS
+------------------------
 
-    That should print several lines including some gibberish at the end.
+1.  **Verify your VO's OIM registration is up-to-date**.  All repositories need to be associated with a VO; the VO needs
+    to make sure an appropriate individual has the _OASIS manager_ role in OIM.  This individual will be responsible
+    for the contents of any of the VO's repositories and will be contacted in case of issues.
```

Comment:
How does one go about becoming an OASIS manager? The wording also leaves something to be desired. Perhaps something like this:

> ll repositories need to be associated with a VO; the VO needs to assign an _OASIS manager_ in OIM who would be responsible for the contents of any of the VO's repositories and will be contacted in case of issues. To designate an OASIS manager, follow the instructions \<link here\>.